Monika After Story/game/script-topics.rpy

```diff
@@ -4940,3 +4940,299 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+    
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_fun_facts_open",category=['misc'],prompt="Fun facts",pool=True))
+
+default fun_facts_started = False
+default persistent._mas_funfactfun = True
+define mas_funfact.fun_count = 20
+define mas_funfact.bad_count = 4
+
+label monika_fun_facts_open:
+    #Determines the fact number and whether it's a bad fact
+    fun_facts_bad_chance = renpy.random.randint(1,100)
+
+    #If player has opened fun facts this session
+    if fun_facts_started:
+        m 1a "Say [player], would you like to hear a fun fact?"
+        m 4a "I've been looking some up just to try and teach us both something new."
+        m 4j "They say you learn something new everyday, this way I'm guaranteeing it!"
+        m 2l "I found most of these online so I can't say they're {i}definitely{/i} true... "
+        $ fun_facts_start = True
+    else:
+        m 1b "Up for another fun fact [player]?"
+        if persistent._mas_funfactfun:
+           m 2j "That last one was pretty interesting after all!"
+        else:
+           m 2m "I know that last one wasn't great...but I'm sure this next one will be better."
+    m 3a "Now, let's see..."
+
+#Determines if it is a bad fact, 10% chance.
+    if fun_facts_bad_chance <= 10:
+        $ jump_fact = "bad_facts_{0}".format(renpy.random.randint(1,4))
```

Comment:
You added constant versions of the literals, which is good, but you didn't put them here. The idea behind this is that if we'd only have to change one line instead of every line the literal is on.